artifacts.go

```diff
@@ -100,6 +100,9 @@ type ArtifactConfig struct {
 	// Name is file or dir name to use for the artifact in the package.
 	// If empty, the file or dir name from the produced artifact will be used.
 	Name string `yaml:"name,omitempty" json:"name,omitempty"`
+	// Mode is the file mode to use for the artifact.
+	// Mode is represented as an fs.FileMode.
+	Mode fs.FileMode `yaml:"mode,omitempty" json:"mode,omitempty"`
```

Comment:
Need to add this to the artifact config info in website/docs.

Reply:
the mode field was already specified in [docs](https://github.com/Azure/dalec/blob/main/website/docs/artifacts.md?plain=1#L42) Is that what you mean?

Reply:
Oh yep, you are correct!